HomeGenie/Service/HomeGenieService.cs

```diff
@@ -1707,7 +1707,23 @@ private void LoadModules()
                 HomeGenie.Service.TsList<Module> modules = (HomeGenie.Service.TsList<Module>)serializer.Deserialize(reader);
                 //
                 foreach (var module in modules)
-                {
+                {                    
+                    if (module.Domain == "HomeAutomation.ConnAir")
```

Comment:
@skl77 this is the old way HG used to create modules. Now this is not used anymore as modules are requested by using the MIGInterface method "GetModules".